File: Core/Stateflows.Common/System/Events/BehaviorInstancesRequest.cs
﻿using Stateflows.Common;
using System.Collections.Generic;

namespace Stateflows.System
{
    public sealed class BehaviorInstancesRequest : Request<BehaviorInstancesResponse>
    {
        public IEnumerable<BehaviorClass> BehaviorClasses { get; set; }
    }
}

File: Transport/@stateflows/common/src/events/behavior-instances.request.ts
import { Request } from "./request";
import { BehaviorInstancesResponse } from "./behavior-instances.response";

export class BehaviorInstancesRequest extends Request<BehaviorInstancesResponse> {
    public $type = "Stateflows.System.BehaviorInstancesRequest, Stateflows.Common";
    public name = "Stateflows.System.BehaviorInstancesRequest";

    constructor() {
        super();
    }
}